src/components/CarouselCard/__tests__/animationButton.spec.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import AnimationButtom from '../animationButton';
+
+describe('<AnimationButtom />', () => {
+    it('should set the right assistive text', () => {
+        const component = mount(<AnimationButtom isAnimationPaused />);
+
+        expect(component.find('span').text()).toBe('Start auto-play');
+    });
+
```

Comment:
add test for when isAnimationPaused is false that the assistiveText passed to ButtonIcon is Stop auto-play.